server/backend.go

```diff
@@ -1,9 +1,12 @@
 package server
 
-// Router paths
 var (
+	// Router paths
 	pathStatus            = "/eth/v1/builder/status"
 	pathRegisterValidator = "/eth/v1/builder/validators"
 	pathGetHeader         = "/eth/v1/builder/header/{slot:[0-9]+}/{parent_hash:0x[a-fA-F0-9]+}/{pubkey:0x[a-fA-F0-9]+}"
 	pathGetPayload        = "/eth/v1/builder/blinded_blocks"
+
+	// Relay Monitor paths
+	pathAuctionTranscript = "/eth/v1/relay-monitor/transcript"
```

Comment:
```suggestion
	pathAuctionTranscript = "/api/v1/relay-monitor/transcript"
```

Reply:
how about `/relay-monitor/v1/transcript`?

Reply:
im open to this -- the idea was to namespace the API of the relay monitor from any other HTTP endpoints that software may host, e.g. landing page

but I'm not super attached to the actual symbol

Reply:
I just thought it'd be cleaner to keep /eth/ for eth specs, like the builder API. Not a strong opinion.

Cc @jtraglia @lightclient @terencechain ?

Reply:
the current thing is `api` and not `eth` -- this doesn't touch any of the "canonical" stuff at all and was just an arbitrary choice I made to namespace these endpoints

and I'm saying it can be `relay-monitor` if you want -- I don't think it is super critical

Reply:
It's not a strong preference, but I prefer `/relay-monitor/v1/transcript` over the other options. Though, I suggest replacing the dash with an underscore (`/relay_monitor/v1/transcript`) as dashes aren't used Ethereum APIs or shortening it to `/monitor/v1/transcript` (this looks cleaner to me).

Reply:
I like it. +1 for `/monitor/`

Reply:
https://github.com/ralexstokes/relay-monitor/pull/33